v2/emailpassword/common-customizations/sessions/update-jwt-payload.mdx

```diff
@@ -414,11 +414,15 @@ func main() {
 		currAccessTokenPayload["newKey"] = "newValue"
 
 		//highlight-next-line
-		err = session.UpdateAccessTokenPayload(handle, currAccessTokenPayload)
+		updated, err := session.UpdateAccessTokenPayload(handle, currAccessTokenPayload)
 		if err != nil {
 			// TODO: handle error
 			return
 		}
+		if !updated {
+			// Session does not exist
+			continue
+		}
```

Comment:
You don't really need this since there is nothing at the end of this loop anyway. Instead, in this loop, you get `sessionInfo` which should be a pointer now, so you need to check if that's nil, and if it is, then you need to call continue.

Reply:
Similar applies to other changes in the PR